src/Catalyst.Core.Lib/Validators/TransactionValidator.cs

```diff
@@ -74,10 +74,8 @@ private bool ValidateTransactionSignature(TransactionBroadcast transactionBroadc
 
             var transactionSignature = _cryptoContext.GetSignatureFromBytes(transactionBroadcast.Signature.RawBytes.ToByteArray(),
                 transactionBroadcast.PublicEntries.First().Base.SenderPublicKey.ToByteArray());
```

Comment:
You must verify the transaction without the signature as no signature was there when the signature was created as you run into a chicken or egg problem.

Reply:
how can you possibly verify a transaction without a signature? the signature must be created at time of creating the transaction